src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const employees = document.querySelectorAll('li');
+
+function sortList(someEmployees) {
+  const sortedEmployees = [...someEmployees].sort((item1, item2) => {
+    const salary1 = +item1.dataset.salary
+      .split(',').join('')
+      .split('$').join('');
```

Comment:
1. Better create helper function for converting `salary` from string to number.

2. Simplify syntax, `split` is not a great option I think :) Suggestions:
- `salary.slice(1).replaceAll(',', '') `
- `salary.replace('$', '').replaceAll(',', '') `